lessons/Data-Science/AI/ai-decision-trees/langs/en-us/index.adoc

```diff
@@ -0,0 +1,335 @@
+
+}
+
+@QandA{
+@Q{After looking at the decision trees of your classmates: What do these trees all have in common? How are they different?}
+@A{Answers will vary. Each tree will have twice as many branches as nodes. Many trees will have the same number of nodes, although probably not all. Many questions asked will likely be the same, but not every question.}
+}
+
+Let's take a step back and see how well some of our decision trees will perform.
+
+@lesson-instruction{
+Complete Q1 on @printable-exercise{comparing-trees.adoc}.
+}
+
+@teacher{
+Invite students to share and explain their responses before emphasizing the main ideas, below.
+}
+
+You just observed how a decision tree can accurately label and categorize the inputs _that it has been trained to label and categorize_. However, the tree we tested—which was 100% accurate on the training dataset—faltered when we offered it inputs that were either *unknown*, like the chopstick, or *ambiguous*, like the spork.
+
+The only way this tree stands a chance of correctly identifying a chopstick or a spork is if we offer it more training!
+
+
+@lesson-instruction{
+Complete the remainder of @printable-exercise{comparing-trees.adoc}.
+}
+
+@QandA{
+@Q{Why is it advantageous for AI to be efficient?}
+@A{Responses will vary, but may include: reduced delays, an improved user experience, greater scalability, decreased environmental impact.}
+@Q{Can you think of any reasons *not* to maximize an AI's efficiency?}
+@A{Responses will vary, but students will likely observe that an increase in efficiency leads to a decrease in accuracy.}
+}
+
+In AI, efficiency and accuracy are often in conflict:
+
+- AI is *efficient* when the computer performs a task with minimal time, memory, energy or data.
+
+- AI is *accurate* when the computer performs its task with correct, relevant, and consistent results.
+
+Striking the perfect balance is an ongoing challenge for computer scientists, and it is a challenge with far-reaching implications.
+
+=== Synthesize
+
+As we built our decision trees, we were able to draw on everything we know about every knife, spoon, spork, plate, bowl or mug that we have ever seen. And our decision tree didn't know about the utensils we either didn't know about yet or forgot to include. If you were asked to create a decision tree to identify common animals or foods, you could probably do that without much difficulty as well. 
+
+Computers build decision trees using only the data we provide... and that data can sometimes be limited or messy. As a result, we may end up with models that are not 100% accurate.
+
+Imagine that you are tasked with building a decision tree that can determine an iris' species (_setosa_, _virginica_, and _versicolor_) based on the varying plant measurements. It would be far more challenging to build this tree than it was to build the common tableware classification tree we just made because... !
+
+@centered-image{images/iris-data.png}
+
```

Comment:
This Synthesize section is not a synthesize section. It doesn't include any questions!